pkgs/applications/version-management/pijul/default.nix

```diff
@@ -1,26 +1,28 @@
 { stdenv
+, darwin
```

Comment:
Please do not use darwin as an input directly.

Reply:
Should I import pkgs then? I am not sure what to do here.

Reply:
The alternative from greping nixpkgs seems to be to import `pkgs` and substitute `darwin` for `pkgs.darwin`.

Reply:
That is even worse. Please do something like
```
a = callPackages ../ {
  inherit (darwin.apple_sdk.frameworks) Security;
};
```

in top-level.

Reply:
Oh ok, yeah that seems better.